README.md

```diff
@@ -219,6 +219,7 @@ masters:
     zmq_auth: True/False	Does the master requires ZMQ authentication.
     zmq_auth_key:		optional path to a public ZMQ key
     zmq_auth_key_secret:	optional path to a private ZMQ key
+    slaves_key:			optional path to a directory with slaves public key. Usefull when you want to create a master without slaves nodes in boards.yaml.
```

Comment:
s/slaves_key/slave_keys/
"optional path to a directory with public key(s) for slave(s).  Useful..."

Also, looks like a whitespace/indent mismatch compared to previous lines.